app/api/v1/entities/person.rb

```diff
@@ -2,17 +2,28 @@ module V1
   module Entities
     class Person < Grape::Entity
       expose :id, documentation: { type: :Integer }
-      expose :url, documentation: { desc: "Canonical URL of the person at Project Ben-Yehuda (useful for giving credit and allowing users to click through)" } do |person|
-        Rails.application.routes.url_helpers.bib_person_url(person)
+      expose :url, documentation: {
+        desc: 'Canonical URL of the person at Project Ben-Yehuda (useful for giving credit and allowing' \
+              'users to click through)'
+      } do |au|
+        Rails.application.routes.url_helpers.bib_authority_url(au)
       end
```

Comment:
Wait, I've only now noticed this URL never pointed to the right place, to begin with: the canonical URL for a person (and now for an authority) should be /author/\d+, i.e. the public table-of-contents view for that person. The bib_person_url is internal, for the few users authorized to use the bibliographical workshop in the back-end.  I missed this while previously reviewing the API, but this field should return authority_url, not bib_authority_url.